### ISSUE TITLE
Products index page is broken

### ISSUE THREAD

**Issue by wonderer007** [thoughtbot/administrate#162]
I just randomly click on http://administrate-prototype.herokuapp.com/admin/products and it is broken. Can I fix this error and submit a patch ?   
Thanks

**Comment by graysonwright** [comment]
@wonderer007 it looks like our recent reorganization of the repo ( #144) has created a few hiccups for the demo site.

If you'd like to take a look at this, that would be great! Otherwise I'll be able to take care of it around 6pm PST.

**Comment by graysonwright** [comment]
Dove into the Heroku logs, found this:

```
2015-11-06T08:41:55.797147+00:00 app[web.1]: Completed 500 Internal Server Error in 7ms (ActiveRecord: 1.0ms)
2015-11-06T08:41:55.808049+00:00 app[web.1]: ** [Airbrake] Notice was not sent due to configuration:
2015-11-06T08:41:55.808051+00:00 app[web.1]:   Environment Monitored? true
2015-11-06T08:41:55.808052+00:00 app[web.1]:   API key set? false
2015-11-06T08:41:55.811532+00:00 app[web.1]:
2015-11-06T08:41:55.811534+00:00 app[web.1]: ActionView::Template::Error (No route matches {:action=>"show", :controller=>"admin/products", :id=>""} missing required keys: [:id]):
2015-11-06T08:41:55.811535+00:00 app[web.1]:     47:       <tr class="table__row"
2015-11-06T08:41:55.811536+00:00 app[web.1]:     48:           role="link"
2015-11-06T08:41:55.811536+00:00 app[web.1]:     49:           tabindex="0"
2015-11-06T08:41:55.811537+00:00 app[web.1]:     50:           data-url="<%= polymorphic_path([Administrate::NAMESPACE, resource]) -%>"
2015-11-06T08:41:55.811538+00:00 app[web.1]:     51:           >
2015-11-06T08:41:55.811538+00:00 app[web.1]:     52:         <% collection_presenter.attributes_for(resource).each do |attribute| %>
2015-11-06T08:41:55.811539+00:00 app[web.1]:     53:           <td class="cell-data cell-data--<%= attribute.html_class %>">
```

**Comment by graysonwright** [comment]
damn, looks like someone added a product where the slug was blank.

**Comment by graysonwright** [comment]
Preventative fix is up at #169.
